src/config/config.ts

```diff
@@ -190,6 +191,76 @@ export function initializeAPIClients() {
     cachePostMethods(GenomeNexusAPI, [], /POST$/);
     cachePostMethods(GenomeNexusAPIInternal, [], /POST$/);
     cachePostMethods(OncoKbAPI);
+
+    compressRequestBodies(
```

Comment:
i think we should say applyGZIPToFetchMothods.  just be super specific about what we are doing in the name of the method.   also, we should comment on the fact that this required server to be specially configured

Reply:
Sure - anything particular you want me to say about the server being specially configured? As long as it's on the right release and has `enable_request_body_gzip_compression` set to true, it will work.